packages/swapper/src/swappers/cow/utils.ts

```diff
@@ -0,0 +1,8 @@
+import { ChainId } from '@shapeshiftoss/caip'
+
+import { Trade } from '../../api'
+import { CowTrade } from './types'
+
+export const isCowTrade = <C extends ChainId>(
+  trade: CowTrade<C> | Trade<C>
+): trade is CowTrade<C> => Boolean((trade as CowTrade<C>).feeAmountInSellToken)
```

Comment:
```suggestion
): trade is CowTrade<C> => Boolean((trade as CowTrade<C>)?.feeAmountInSellToken)
```

without pulling locally it feeeeels like we'll need opt chain here?

Reply:
also curious if the `in` syntax works.

`'feeAmountInSellToken' in trade`